package.json

```diff
@@ -81,7 +83,7 @@
     "prettier:write": "yarn prettier --write $npm_package_config_prettier_target",
     "eslint:check": "eslint $npm_package_config_prettier_target --max-warnings=0",
     "validate": "yarn typescript:check && yarn prettier:check && yarn eslint:check",
-    "dev": "yarn typescript:watch && yarn storybook:dev",
+    "dev": "yarn typescript:watch & yarn storybook:dev",
```

Comment:
This didn't yield so it just hung.

Reply:
do we need to run typescript watch when running storybook? could we not use the source files for the storybook addons?

iirc there could be collisions with watchers that might make this.. not work great

Reply:
I'll need to verify this. My guess is that storybook wasn't picking up all TS errors correctly or in a readable format. 

It works well at the moment, but I understand what you're saying.

Reply:
Ok so I have an answer here:

`tsc` is for building the addon itself. yarn `storybook:dev` is for building the storybook stories. Both are built and recompiled separately.